apps/wallet/src/components/inputs/StandardsAutocomplete.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <VCombobox
+    v-model="model"
+    :multiple="props.multiple.value"
+    :label="props.label.value"
+    item-value="value"
+    item-title="text"
+    :items="items"
+    :variant="props.variant.value"
+    :density="props.density.value"
+    :readonly="props.readonly.value"
+    :disabled="props.disabled.value"
+    :rules="props.rules.value"
+  />
+</template>
+
+<script setup lang="ts">
+import { computed, toRefs } from 'vue';
+import { useI18n } from 'vue-i18n';
+import { VCombobox } from 'vuetify/components';
+import { useStationStore } from '~/stores/station.store';
+import { FormValidationRuleFn } from '~/types/helper.types';
+
+const input = withDefaults(
+  defineProps<{
+    modelValue?: string[];
+    blockchain: string;
+    label?: string;
+    variant?: 'underlined' | 'outlined' | 'filled';
+    density?: 'comfortable' | 'compact';
+    multiple?: boolean;
+    readonly?: boolean;
+    disabled?: boolean;
+    rules?: FormValidationRuleFn[];
+  }>(),
+  {
+    modelValue: undefined,
+    label: undefined,
+    variant: 'filled',
+    density: 'comfortable',
+    multiple: false,
+    readonly: false,
+    disabled: false,
+    rules: undefined,
+  },
+);
+
+const props = toRefs(input);
```

Comment:
nit: i think we were doing this in some other places, but we should be able to use the props directly without the need to wrap it in `toRefs`